app.rb

```diff
@@ -0,0 +1,200 @@
+require 'sinatra'
+require 'killbill_client'
+require 'dwolla_v2'
+
+set :kb_url, ENV['KB_URL'] || 'http://127.0.0.1:8080'
+set :client_id, ENV['DWOLLA_CLIENT_ID']
+set :client_secret, ENV['DWOLLA_CLIENT_SECRET']
+set :dwolla_access_token, ENV['DWOLLA_ACCESS_TOKEN']
+
+#
+# Kill Bill configuration and helpers
+#
+
+KillBillClient.url = settings.kb_url
+
+# Multi-tenancy and RBAC credentials
+options = {
+    :username => 'admin',
+    :password => 'password',
+    :api_key => 'bob',
+    :api_secret => 'lazar'
+}
+
+# Audit log data
+user = 'demo'
+reason = 'New subscription'
+comment = 'Trigger by Sinatra'
+
+def create_kb_account(user, reason, comment, options)
+  account = KillBillClient::Model::Account.new
+  account.name = 'John Doe'
+  account.currency = 'USD'
+  account.create(user, reason, comment, options)
+end
+
+def create_kb_payment_method(account, funding_source, customer_id, user, reason, comment, options)
+  puts "Customer ID: #{customer_id}"
```

Comment:
Debug? Having these is fine, but I would suggest using a real logger instead (Sinatra may offer one out of the box already, TBD).